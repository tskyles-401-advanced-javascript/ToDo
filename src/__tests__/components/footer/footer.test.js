import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import React from 'react';
import renderer from "react-test-renderer";
import Footer from '../../../components/footer/footer';
Enzyme.configure({ adapter: new Adapter() });

describe('<Footer />', () => {
  it('is alive on start', () => {
    let app = shallow(<Footer/>);

    expect(app.find('footer').exists()).toBeTruthy();
    expect(app.find('p').exists()).toBeTruthy();

  });

  it('renders correctly', () => {
    const tree = renderer.create(<Footer/>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});