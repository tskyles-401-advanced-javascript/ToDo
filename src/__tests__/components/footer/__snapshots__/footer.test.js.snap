// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<footer>
  <p>
    Created by Travis Skyles
  </p>
</footer>
`;
